wiki/Rules/id.md

```diff
@@ -18,7 +18,7 @@ Peraturan-peraturan berikut ini merupakan pedoman dasar yang kami terapkan di li
 3. **Jika di tengah pembicaraan terdapat seseorang yang terputus dari jaringan *(timed out)*, di-silence, atau di-restrict, abaikan saja dan jangan mulai membicarakannya.** Jika seseorang di-restrict, kasus mereka akan otomatis jatuh pada ranah yang hanya melibatkan staf administratif osu! dan diri mereka sendiri.
 4. **Selalu jaga norma sportivitas dalam permainan, dan jangan indahkan para pemain yang berbuat curang atau melakukan tindakan-tindakan yang tidak terpuji lainnya.** Sayangnya orang-orang seperti ini akan selalu ada dari waktu ke waktu, dan pada umumnya semakin mereka diperhatikan semakin keonaran mereka akan menjadi-jadi. Laporkan mereka melalui perintah `!report` atau tombol Report pada laman profil mereka dan biarkan semuanya berlalu.
 5. **Jaga topik obrolan pada kanal publik agar tetap sesuai bagi segala usia dan hindari topik NSFW apabila memungkinkan.** Hal-hal yang mengandung unsur-unsur NSFW tidak sejalan dengan arah tujuan komunitas osu! yang terbuka bagi semua usia. Kamu bebas untuk mengirimkan segala hal yang kamu kehendaki dalam pesan-pesan pribadi (selama tidak melampaui batas), namun topik obrolan pada kanal-kanal publik harus selalu dijaga agar dapat dengan nyaman dinikmati oleh semua orang.
-6. **Bersikaplah bijak seputar hal-hal yang hendak kamu bagikan kepada orang lain, terutama jika hal-hal tersebut menyangkut konten yang dapat mengganggu kenyamanan umum.** Apabila kamu harus berpikir lebih dari satu kali sebelum mengimbuhkan konten yang ingin kamu kirim, besar kemungkinan konten tersebut memang kurang layak untuk dikirim. Pada umumnya, sebisa mungkin hindari mengirim hal-hal yang dapat memicu epilepsi, mengandung unsur kekerasan, atau konten-konten sensitif lainnya ke dalam ruang publik.
+6. **Bersikaplah bijak seputar hal-hal yang hendak kamu bagikan kepada orang lain, terutama apabila hal-hal tersebut menyangkut konten-konten yang dapat mengganggu kenyamanan umum.** Apabila kamu harus berpikir lebih dari satu kali sebelum mengimbuhkan konten yang ingin kamu kirim, besar kemungkinan konten tersebut memang kurang layak untuk dikirim. Pada umumnya, sebisa mungkin hindari mengirim hal-hal yang dapat memicu epilepsi, mengandung unsur kekerasan, atau konten-konten sensitif lainnya ke dalam ruang publik.
 7. **Jaga tautan hanya pada situs-situs web yang berstatus legal.** Distribusi konten-konten video (di samping melalui situs-situs web yang telah berlisensi resmi seperti Animelab dan Crunchyroll) atau perangkat lunak/"warez" milik pihak lain secara ilegal dalam bentuk apapun tidak dibenarkan di dalam lingkungan osu!. Tautan siaran Twitch non-osu! HANYA dibenarkan untuk disebar pada kanal `#videogames`.
```

Comment:
```suggestion
7. **Jaga tautan hanya pada situs-situs web yang berstatus legal.** Distribusi konten-konten video (di samping melalui situs-situs web yang telah berlisensi resmi seperti Animelab dan Crunchyroll) atau perangkat lunak/"warez" milik pihak lain secara ilegal dalam bentuk apapun tidak dibenarkan di dalam lingkungan osu!. Tautan siaran Twitch yang diluar pada osu! HANYA boleh disebar pada kanal `#videogames`.
```

Reply:
changed this one line into ``Adapun tautan yang tertuju pada kanal Twitch yang menayangkan konten-konten yang tidak berkaitan dengan osu! HANYA boleh disebar pada kanal #videogames.``